src/usr/local/www/vpn_ipsec_phase1.php

```diff
@@ -732,7 +732,8 @@ function build_eal_list() {
 		'*Remote Gateway',
 		'text',
 		$pconfig['remotegw']
-	))->setHelp('Enter the public IP address or host name of the remote gateway.');
+	))->setHelp('Enter the public IP address or host name of the remote gateway. ' .
+		    'Use 0.0.0.0 to allow connection from any IPv4 address.');
```

Comment:
Needs more explanation about the limitations of doing that: Can't initiate, can't use the peer IP address for the remote identifier, and may be limited to a single entry.

Though the only one entry may have been address by PR #4173 but I don't know if anyone tested that with `0.0.0.0`. Test to confirm that.

The identifier/PSK issue may have a workaround (define a PSK for `%any` on the PSK tab) but that's probably not something we should encourage.

Reply:
> Needs more explanation about the limitations of doing that: Can't initiate, can't use the peer IP address for the remote identifier, and may be limited to a single entry.

Added

> Though the only one entry may have been address by PR #4173 but I don't know if anyone tested that with `0.0.0.0`. Test to confirm that.

Multiple entries with '0.0.0.0' with different IDs works fine

> The identifier/PSK issue may have a workaround (define a PSK for `%any` on the PSK tab) but that's probably not something we should encourage.

No issues were found after adding Mobile IPsec with PSK% any - other site-to-site '0.0.0.0' connections successfully connect